wshop-core.php

```diff
@@ -1,6 +1,51 @@
 <?php
 
 /*
+ * Add Custom Post Type 'Product'
+ */
+    function wshop_create_custom_post() {
+
+        $labels = array(
+            'name'                  => 'Products',
+            'all_items'             => 'All Products',
+            'singular_name'         => 'Product',
+            'add_new'               => 'Add New Product',
+            'add_new_item'          => 'Add New Product',
+            'edit_item'             => 'Edit Product',
+            'new_item'              => 'New Product',
+            'view_item'             => 'View Product',
+            'search_items'          => 'Search Products',
+            'not_found'             => 'No products found',
+            'not_found_in_trash'    => 'No products found in Trash'
+        );
+
+        $args = array(
+            'labels'                => $labels,
+            'public'                => true,
+            'publicly_queryable'    => true,
+            'capability_type'       => 'page',
+            'menu_icon'             => 'dashicons-cart',
+            'hierarchical'          => true,
```

Comment:
Not actually sure that we need this now that I'm thinking about it. Let's talk about it